docs/application/web/guides/data/file-system.md

```diff
@@ -108,149 +107,261 @@ To manage file storages:
 
-## Retrieving Files and File Details
+## Deleting Files
 
-You can retrieve a list of files or file URIs using the `listFiles()` and `toURI()` methods. The URI can be used to identify the file, for example, by using it as the `src` attribute on an HTML `img` element.
+You can create directories using the `createDirectory()` method. The directory is created at `path` specified in parameter.
+You can delete files using `deleteFile()` method. Target pointed by `path` specified will be deleted from filesystem.
 
-You can retrieve file content as a `DOMString` with the `readAsText()` method. The encoding input parameter of the method defines the format in which the file content is returned.
-
-To get files and file details from the file system:
-
-1. To access a specific file or directory within the file system, retrieve a file handle using the `resolve()` method of the `FileSystemManager` interface (in [mobile](../../../../org.tizen.web.apireference/html/device_api/mobile/tizen/filesystem.html#FileSystemManager), [wearable](../../../../org.tizen.web.apireference/html/device_api/wearable/tizen/filesystem.html#FileSystemManager), and [TV](../../../../org.tizen.web.apireference/html/device_api/tv/tizen/filesystem.html#FileSystemManager) applications):
+1. To delete a file, use the `deleteFile()` method:
 
    ```
-   tizen.filesystem.resolve('documents', onResolveSuccess, null, 'r');
+   var fullPath = 'documents/file_to_be_deleted.txt'
+   var callback = function(modifiedDirectory) {
+       console.log('deleteFile() is successfully done. Modified (parent) directory: ' + modifiedDirectory);
+   }
+   var errorCallback = function(error) {
+       console.log(error);
+   }
+   tizen.filesystem.deleteFile(fullPath, callback, errorCallback);
    ```
 
-   The `File` object (in [mobile](../../../../org.tizen.web.apireference/html/device_api/mobile/tizen/filesystem.html#File), [wearable](../../../../org.tizen.web.apireference/html/device_api/wearable/tizen/filesystem.html#File), and [TV](../../../../org.tizen.web.apireference/html/device_api/tv/tizen/filesystem.html#File) applications) is returned in the success event handler.
+## Retrieving Files and File Details
 
-2. To retrieve a list of all the files and their directories located in a specified directory, use the `listFiles()` method of the `File` object:`function onResolveSuccess(dir) {    dir.listFiles(onsuccess);}`The method returns an array of `File` objects.
+You can retrieve a list of files or file URIs using the `listDirectory()` and `toURI()` methods. The URI can be used to identify the file, for example, by using it as the `src` attribute on an HTML `img` element.
 
-3. To retrieve the file URI, use the `toURI()` method:
+1. To retrieve a list of all files and directories located in a specified directory, use the `listDirectory()` method of the `filesystem` object. The method returns an array of `DOMString` elements, each beeing a path with name of file inside a directory pointed by `path`.
```

Comment:
each beeing

Please rephrase to each being.

Reply:
Done.